website/views/shared/tasks/meta_controls.jade

```diff
@@ -2,11 +2,11 @@
 
   // Counter
   span(ng-if='showDoubleTaskCounter(task, obj)')
-    span(tooltip=env.t('habitCounterUp')) +{{task.counterUp}}|
-    span(tooltip=env.t('habitCounterDown')) -{{task.counterDown}}&nbsp;
+    span(tooltip='{{env.t("habitCounterUp")}} ({{env.t("resets")}} {{env.t(task.frequency)}})') +{{task.counterUp}}|
```

Comment:
Go ahead and submit a fix PR and we'll roll it in before the main PR goes into production!